spec/rails_helper.rb

```diff
@@ -55,3 +55,31 @@
   config.include FactoryGirl::Syntax::Methods
 
 end
+
+Geocoder.configure(lookup: :test)
```

Comment:
Can you add a small comment here about why you added this? (it's to fix some geocoding issue, right?)